doc/ReleaseNotes/_ReleaseNotes.md

```diff
@@ -1,11 +1,11 @@
 ﻿# Release notes
 
 ### Features
+- Support for `EmailManager.ShowComposeNewEmailAsync`
 - Add support for Automation SetDependencyPropertyValue in Uno.UITest
 - Added support for using a `string` value in a `StaticResource` when using `CreateFromStringAttribute'
 - [Android] Adds support for `FeatureConfiguration.ScrollViewer.AndroidScrollbarFadeDelay`
-- Add support for `Grid.ColumnSpacing` and `Grid.RowSpacing`
-- Add clarification in [documentation](../articles/uno-development/working-with-the-samples-apps.md) for adding automated UI tests
+- [#2040] Support for ms-settings:// special URIs on Android and iOS, Launcher API alignments to match UWP behavior
```

Comment:
Don't remove release notes ;)

Reply:
Oh! That was definitely not intentional 😅